### ISSUE TITLE
[Feature Request] Change keybindings

### ISSUE THREAD

**Issue by TornaxO7** [logisim-evolution/logisim-evolution#1455]
Hello! I couldn't find a way to change the default keybindings for example to change to "Edit selection and wires"-mode.
It would be nice to map them to something else than using `Ctrl`+`<num>` :)

**Comment by MarcinOrlowski** [comment]
Keybindings are currently not configurable.

**Comment by hewzhew** [comment]
I investigated this against the current `main`. PR #1802 made many shortcuts configurable, but the ten toolbar-selection shortcuts are still hard-coded in `KeyboardToolSelection` as the platform menu modifier plus `1`–`9`, `0`.

I propose a focused update that exposes those existing ten selectable toolbar slots in **Preferences → Hotkeys**, while preserving the current platform-specific defaults. The bindings would update open windows immediately, participate in the existing conflict checks/reset flow, and appear correctly in toolbar tooltips. Selection would keep its current slot semantics (including skipping separators), so customized toolbar ordering remains respected.

I can implement this with focused regression coverage if that scope looks right.

**Comment by maehne** [comment]
I agree, but would not limit it to the first 10 toolbar icons, but also would offer to configure shortcuts for those, which currently don't have one assigned (and by default should be left unassigned).

IMHO, the usability of the toolbar could be improved. Only the first 6 entries, I use regularly. Between the hand tool and selection tool, I typically switch using the keyboard shortcut. It might make sense to make the toolbar configurable as in many modern GUI applications, but this is outside the scope of this issue.
